processor/schemaprocessor/internal/migrate/migrator.go

```diff
@@ -0,0 +1,16 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package migrate // import "github.com/open-telemetry/opentelemetry-collector-contrib/processor/schemaprocessor/internal/migrate"
+
+// Migrator is an interface that all migration types must implement.  It is basically a marker interface.  All Operators are also Migrators
+type Migrator interface {
```

Comment:
That's an idea!  Right now I use Operator and Migrator... but Transformer and Migrator might be better.  I also am kind of considering `Change`?  Since the yaml key is called `changes`.  Thoughts?

Reply:
Went with Transformer